start.js

```diff
@@ -0,0 +1,51 @@
+const http = require('http');
+const soldiers = require('./routs/soldiers.js');
+const duties = require('./routs/duties.js');
+const justiceBoard = require('./routs/justiceBoard.js');
+
+function runServer() {
+  let server = http.createServer(function (req, res) {
+    let body = [];
+    req.on('data', (chunk) => {
+      body.push(chunk);
+    }).on('end', () => {
+      body = Buffer.concat(body).toString();
+      if (body !== "") {
+        body = JSON.parse(body);
+      }
+      const parsedUrl = req.url.split("/");
+      if (/soldiers.*/.test(parsedUrl[1]) && parsedUrl.length < 4) {
```

Comment:
Why the length limit here?
If you're adding here the knowledge of what's the maximum length a valid call could have, remove it from here and move it to each separate file instead. Otherwise you just added another place that needs to be checked and edited when adding a new route or removing one.